README.md

```diff
@@ -116,8 +110,66 @@ Example command to generate `pallet_fees` with default `output`
 default output will be `./pallets/fees/src/weight.rs`
 You can override this by passing output path as last argument
 
-## Upgrading to latest cumulus(until they have tags for releases)
-1. First collect commits of Substrate, Grandpa-Bridge, Polkadot from the latest cumulus
-2. Bring our fork of all the above repos to the above commits
-3. Use diener in Grandpa brigde, polkadot, and cumulus to use our forks in `centrifuge` branch
-4. Then update deps on centrifuge chain
+### Benchmarking runtimes
+
+When benchmarking pallets, we are just running the benchmarking scenarios they specify
+within their mocked runtime. This fails to actually benchmark said pallet in the context
+in which it will be actually used in production: within a specific runtime and composing
+with other pallets.
+
+To cover that, we run test for every pallet for a given runtime and use the output weights
+in production since those are the most trustworthy weights we can use.
+
+Note: This command should be run in a cloud environment that mimics closely the specs of
+the collator node the parachain will be running on.
+
+```shell
+./scripts/runtime_benchmarks.sh <runtime>
+```
+
+# Upgrading to a newer Polkadot / Substrate / Cumulus version
+
+When a new version of Polkadot is released, a mirroring release hapens for the other 
+parity projects such as Substrate and Cumulus, but also for other third-party projects
+such as the `orml` pallets, `xcm-simulator`, etc.
+
+You can follow these steps tailored to update only one of those specific projects
+updating the three simultaneously is the most common flow.
+
+The flow to upgrade to a newer version of polkadot is usually as follows:
+
+1. Use `diener` to update Polkadot, Substrate, and Cumulus to said release
```

Comment:
Probably makes sense to add a link to the tool: https://github.com/bkchr/diener right?

Reply:
Thanks for the review and the suggestion, fixed 👍